go.mod

```diff
@@ -9,8 +9,12 @@ require (
 	github.com/prometheus/common v0.2.0
 	github.com/prometheus/procfs v0.0.0-20190209105433-f8d8b3f739bd // indirect
 	github.com/stretchr/testify v1.3.0 // indirect
+	golang.org/x/net v0.0.0-20190522155817-f3200d17e092 // indirect
 	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
+	golang.org/x/sys v0.0.0-20190531175056-4c3a928424d2 // indirect
```

Comment:
`golang.org/x/sys` isn't used directly or indirectly by `promu` so it gets removed when running `go mod tidy`.